jsr.json

```
@@ -0,0 +1,14 @@
+{
+  "name": "@peterhirn/test",
```

```suggestion
  "name": "@phi-ag/argon2",
```

Why not having the samething that npm ?